docker/files/build/install.sh

```diff
@@ -82,7 +82,11 @@ fi
 git log -n 5
 
 ./scripts/configure_dev.sh
-make build
-./scripts/dev_install.sh -p "${BINDIR}" -d "${ALGORAND_DATA}"
+# make sure the makefile uses specific values for BUILD_NUMBER and BRANCH
+BUILD_NUMBER="" BRANCH="$BRANCH" make build
+
+shopt -s extglob
+
+cd "$BINDIR" && rm -vrf !(algocfg|algod|algoh|algokey|carpenter|catchupsrv|ddconfig.sh|diagcfg|find-nodes.sh|goal|kmd|msgpacktool|node_exporter|tealcut|tealdbg|update.sh|updater|COPYING)
```

Comment:
I think we could remove all of these without any controversy:
```suggestion
cd "$BINDIR" && rm -vrf !(algocfg|algod|algokey|diagcfg|goal|kmd|msgpacktool|node_exporter|tealdbg|update.sh|updater|COPYING)
```

We could probably even remove algokey / updater / update.sh -- I'd leave them in only because there might be oddball use cases where you pull this image just to run algokey or update.sh.